docs-src/ref-programs.scrbl

```diff
@@ -1890,3 +1890,73 @@ of iterations the algorithm should perform.
 For reference, when performing @reachin{5} iterations, the algorithm can reliably calculate the square root
 up to @tt{32} squared, or @tt{1,024}. When performing @reachin{10} iterations, the algorithm can reliably calculate the
 square root up to @tt{580} squared, or @tt{336,400}.
+
+@subsubsection{@tt{pow}}
+
+@(mint-define! '("pow"))
+@reach{
+  pow (2, 40, 10) // => 1,099,511,627,776 }
+
+@index{pow} @reachin{pow(base, power, precision)} Calculates the approximate value of raising base to power.
+The third argument must be an @reachin{UInt} whose value is known at compile time, which represents the number
+of iterations the algorithm should perform. For reference, @tt{10} iterations provides accuracy until the
+point of overflow.
+
+@subsubsection{@tt{Fixed-Point Numbers}}
+
+@reachin{FixedPoint} is defined by
+@reach{
+  export const FixedPoint = Object({ scale: UInt, i: UInt }); }
+
+@reachin{FixedPoint} can be used to represent numbers with a fixed number of digits after the decimal point.
+They are handy for representing fractional values, especially in base 10. The value of a fixed point number is determined
+by dividing the underlying integer value, @tt{i}, by its scale factor, @tt{scale}. For example, we could
+represent the value @reachin{1.234} with @reachin{{ scale: 1000, i : 1234 }} or @reachin{mk_fixed_point(1000)(1234)}.
+A scale factor of @tt{1000} correlates to 3 decimal places of precision. Similarly, a scale factor of @tt{100} would
+have 2 decimal places of precision.
+
+@(mint-define! '("mk_fixed_point"))
+@reach{
+  const scale = 10;
+  const i = 56;
+  mk_fixed_point(scale)(i) // represents 5.6 }
+
+@index{mk_fixed_point} @reachin{mk_fixed_point(scale)(i)} will return a function that can be used to
+instantiate fixed point numbers with a particular scale factor.
+
+@(mint-define! '("fixed_point_convert_scale"))
+@reach{
+  const x = mk_fixed_point(1000)(1234); // x = 1.234
+  fixed_point_convert_scale(x, 100);    // => 1.23 }
+
+@index{fixed_point_convert_scale} @reachin{fixed_point_convert_scale(x, scale)} will convert a fixed point number from using
+one scale to another. This operation can result in loss of precision, as demonstrated in the above example.
+
+@(mint-define! '("fixed_point_homogenize_scales"))
```

Comment:
This is a descriptive name and I hate it. I want to say `fxmeet` but that's a bad (too technical name), please think of something else :)

Reply:
This one took me a while to find a suitable name 😆 I'll rename